composer.json

```diff
@@ -20,7 +20,8 @@
     "nunomaduro/collision": "^6.4",
     "php-open-source-saver/jwt-auth": "^2.2",
     "psr/simple-cache": "^3.0",
-    "ronasit/laravel-helpers": "dev-master",
+    "ronasit/laravel-helpers": "^2.5",
+    "ronasit/laravel-media": "^0.0.1@beta",
```

Comment:
we no need to media module installed by default, let's remove it

Reply:
Done. Completely removed Media from the project.